nvme-print-stdout.c

```diff
@@ -2167,19 +2167,25 @@ static void stdout_id_ctrl_cqes(__u8 cqes)
 static void stdout_id_ctrl_oncs(__le16 ctrl_oncs)
 {
 	__u16 oncs = le16_to_cpu(ctrl_oncs);
-	__u16 rsvd = (oncs & 0xFE00) >> 9;
-	__u16 copy = (oncs & 0x100) >> 8;
-	__u16 vrfy = (oncs & 0x80) >> 7;
-	__u16 tmst = (oncs & 0x40) >> 6;
-	__u16 resv = (oncs & 0x20) >> 5;
-	__u16 save = (oncs & 0x10) >> 4;
-	__u16 wzro = (oncs & 0x8) >> 3;
-	__u16 dsms = (oncs & 0x4) >> 2;
-	__u16 wunc = (oncs & 0x2) >> 1;
-	__u16 cmp = oncs & 0x1;
+	__u16 rsvd = oncs >> 11;
+	__u16 afc  = oncs >> 10 & 1;
+	__u16 csa  = oncs >> 9 & 1;
+	__u16 copy = oncs >> 8 & 1;
+	__u16 vrfy = oncs >> 7 & 1;
+	__u16 tmst = oncs >> 6 & 1;
+	__u16 resv = oncs >> 5 & 1;
+	__u16 save = oncs >> 4 & 1;
+	__u16 wzro = oncs >> 3 & 1;
+	__u16 dsms = oncs >> 2 & 1;
+	__u16 wunc = oncs >> 1 & 1;
+	__u16 cmp  = oncs & 1;
```

Comment:
Are there libnvme definitions for these?

Reply:
Yeah, I think so. While we're here, we might as well change this to use those for all of them.

Reply:
```
nvme id-ctrl /dev/nvme0 -H | grep -A 11 "All Fast Copy"
  [10:10] : 0   All Fast Copy Not Supported
  [9:9] : 0     Copy Single Atomicity Not Supported
  [8:8] : 0     Copy Not Supported
  [7:7] : 0     Verify Not Supported
  [6:6] : 0     Timestamp Not Supported
  [5:5] : 0x1   Reservations Supported
  [4:4] : 0     Save and Select Not Supported
  [3:3] : 0x1   Write Zeroes Supported
  [2:2] : 0x1   Data Set Management Supported
  [1:1] : 0     Write Uncorrectable Not Supported
  [0:0] : 0x1   Compare Supported

```